poucave/html/js/app.js

```diff
@@ -246,12 +249,23 @@ class Overview extends Component {
           <div class="card-body text-center">
             <i class="fa fa-4x ${iconClass}"></i>
             <p>
-              <strong>The current system status is ${isHealthy ? "Healthy" : "Unhealthy"}.</strong>
+              <strong>The system ${isHealthy ? "is currently healthy" : "has failing checks"}.</strong>
               <br />
               <span class="text-gray-medium">
                 Last updated <${TimeAgo} date="${new Date()}" />.
               </span>
             </p>
+            <p>
```

Comment:
this should be a `ul` not a `p`. or there should be a `ul` inside this `p` tag to wrap the `li` nodes.

Reply:
also i was probably give it a class of `text-red`